system-addon/test/functional/mochitest/head.js

```diff
@@ -115,13 +113,9 @@ async function check_highlights_elements(selector, length, message) { // eslint-
   let browser = gBrowser.selectedBrowser;
   await waitForPreloaded(browser);
 
-  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", false);
-  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", true);
-
   await BrowserTestUtils.waitForCondition(() => content.document.querySelector(selector), `No elements matching ${selector} were found`);
 
-  let found = await ContentTask.spawn(browser, selector, arg =>
-    content.document.querySelectorAll(arg).length);
+  const found = await ContentTask.spawn(browser, selector, arg => content.document.querySelectorAll(arg).length);
   ok(found === length, `there should be ${length} of ${selector} found ${found}`);
```

Comment:
It's probably better to use `is` as when `ok` fails, it'll just say it wasn't truthy. Although I suppose you at least include the values in the printed message.